lib/xpm_ruby/schema/client/contact/add.rb

```diff
@@ -9,7 +9,7 @@ module Contact
             ID: Types::Coercible::String
           ).with_key_transform(&:to_sym),
           Name?: Types::String,
-          IsPrimary?: Types::Bool,
+          IsPrimary?: Types::String,
```

Comment:
noticed value needs to be "YES" or "NO" so I don't think the bool type will work...

Reply:
👍

Reply:
Yeah, I set it to string on my schemas too